edx_proctoring/views.py

```diff
@@ -168,6 +170,61 @@ def handle_exception(self, exc):
         return resp
 
 
+class ProctoredExamAttemptsMFEView(ProctoredAPIView):
+    """
+    Endpoint for getting timed or proctored exam and its attempt data.
+    /edx_proctoring/v1/proctored_exam/exam_attempts/course_id/{}/content_id/(?P<content_id>{})
+
+    Supports:
+        HTTP GET: Returns an existing exam (by course_id and content id) with latest related attempt.
+        and active attempt for any exam in progress (for the timer feature)
+        {
+            'exam': {
+                ...
+                attempt: { ... }
+            },
+            'active_attempt': { ... },
+        }
+    """
+
+    def get(self, request, course_id, content_id):
+        """
+        HTTP GET handler. Returns exam with attempt and active attempt
+        """
+        active_attempt_data = {}
+        attempt_data = {}
+
+        active_exams = get_active_exams_for_user(request.user.id)
+        if active_exams:
+            active_exam_info = active_exams[0]
+            active_exam = active_exam_info['exam']
+            active_attempt = active_exam_info['attempt']
+            active_attempt_data = get_exam_attempt_data(
+                active_exam.get('id'),
+                active_attempt.get('id'),
+                is_learning_mfe=True
+            )
+        try:
+            exam = get_exam_by_content_id(course_id, content_id)
```

Comment:
I would avoid the dublication of data collection in case active axam and exam are equal.

Reply:
adressed, please check